requirements.txt

```diff
@@ -1,6 +1,22 @@
+atomicwrites==1.3.0
```

Comment:
also, just remove packages you haven't by added yourself.
Only add the necessary ones